test/videojs-contrib-hls.test.js

```diff
@@ -692,18 +697,16 @@ QUnit.test('selects a playlist below the current bandwidth', function(assert) {
 
   // the default playlist has a really high bitrate
   this.player.tech_.hls.playlists.master.playlists[0].attributes.BANDWIDTH = 9e10;
-  // playlist 1 has a very low bitrate
-  this.player.tech_.hls.playlists.master.playlists[1].attributes.BANDWIDTH = 1;
   // but the detected client bandwidth is really low
-  this.player.tech_.hls.bandwidth = 10;
+  this.player.tech_.hls.bandwidth = 1;
 
   playlist = this.player.tech_.hls.selectPlaylist();
   assert.strictEqual(playlist,
-                     this.player.tech_.hls.playlists.master.playlists[1],
+                     this.player.tech_.hls.playlists.master.playlists[2],
```

Comment:
I believe the title of this test is no longer accurate. Maybe `selects a playlist below the current moving average bandwidth`